src/Nethermind/Nethermind.Network/Metrics.cs

```diff
@@ -156,6 +156,24 @@ public static class Metrics
         [Description("Number of eth.65 NewPooledTransactionHashes messages sent")]
         public static long Eth65NewPooledTransactionHashesSent { get; set; }
 
+        [Description("Number of eth.68 NewPooledTransactionHashes messages received")]
+        public static long Eth68NewPooledTransactionHashesReceived { get; set; }
+
+        [Description("Number of eth.68 NewPooledTransactionHashes messages sent")]
+        public static long Eth68NewPooledTransactionHashesSent { get; set; }
+
+        [Description("Number of eth NewPooledTransactionHashes messages received")]
+        public static long EthNewPooledTransactionHashesReceived
+        {
+            get => Eth68NewPooledTransactionHashesReceived + Eth65NewPooledTransactionHashesReceived;
```

Comment:
NP: probably should be done in grafana via query instead as extra metrics cost money.

Reply:
Do we want different metrics for eth68 and eth65?

Reply:
Don't know. But we already have them separately here.